RecoPixelVertexing/PixelLowPtUtilities/python/ClusterShapeHitFilterESProducer_cfi.py

```diff
@@ -11,6 +11,5 @@
 )
 from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
 phase2_tracker.toModify(ClusterShapeHitFilterESProducer,
-    PixelShapeFile = 'RecoPixelVertexing/PixelLowPtUtilities/data/pixelShape_Phase1Tk.par',
-    doPixelShapeCut = cms.bool(False)
+    PixelShapeFile = 'RecoPixelVertexing/PixelLowPtUtilities/data/pixelShape_Phase2Tk.par',
```

Comment:
The addition of multiple phase2_tracker Eras is possible, but introduces a lot of complexity. If @ebrondol can show the performance is not too bad, it would be easier just to use the same pixel shape everywhere. (@makortel: moving to GT would imply we'd have to create a separate GT queue for each Phase2 detector version... also not ideal.)